modules/lb/variables.tf

```diff
@@ -1,4 +1,5 @@
 variable "env_code" {}
 variable "lb_subnets" {}
 variable "vpc_id" {}
+variable "certificate" {}
```

Comment:
variable certificate is not defined in the parent lb module as an attribute of the module.